mappings/net/minecraft/client/resource/VideoWarningManager.mapping

```diff
@@ -10,7 +10,7 @@ CLASS net/minecraft/class_5407 net/minecraft/client/resource/VideoWarningManager
 		ARG 0 array
 		ARG 1 patterns
 	METHOD method_30060 getRendererWarning ()Ljava/lang/String;
-	METHOD method_30061 loadWarnlist (Lnet/minecraft/class_3300;Lnet/minecraft/class_3695;)Lcom/google/gson/JsonObject;
+	METHOD method_30061 loadWarnList (Lnet/minecraft/class_3300;Lnet/minecraft/class_3695;)Lcom/google/gson/JsonObject;
```

Comment:
This is called `gpu_warnlist.json`, where the snake case suggest "warnlist" is a whole thing I guess?

Reply:
Agreed, this seems to be along the lines of whitelist which remains a single word:

https://github.com/FabricMC/yarn/blob/097b138f6226efaf4c308e6334cf76da8a163839/mappings/net/minecraft/server/PlayerManager.mapping#L11